openshift_pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -0,0 +1,72 @@
+[id='creating-applications-with-cicd-pipelines']
+= Creating applications with OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+:context: creating-applications-with-cicd-pipelines
+toc::[]
+
+With {pipelines-title}, you can create a customized CI/CD solution to build, test, and deploy your application.
+
+To create a full-fledged, self-serving CI/CD Pipeline for an application, you have to perform the following tasks:
+
+* Create custom Tasks, or install existing reusable Tasks.
+* Create a Pipeline and PipelineResources to define the delivery Pipeline for your application.
+* Create a PipelineRun to instantiate and invoke the Pipeline.
+* Add Triggers to capture any events in the source repository.
+
+This section uses the `pipelines-tutorial` example to demonstrate the above tasks. The example uses a simple application which consists of:
+
+* A front-end interface `vote-ui`, with the source code in link:https://github.com/openshift-pipelines/vote-ui/tree/release-tech-preview-1[`ui-repo`] Git repository.
```

Comment:
Just double checking that you've gotten approval for these GH repos. I see you did get it in another PR:
https://github.com/openshift/openshift-docs/pull/20757#discussion_r411055195

Reply:
Yes,  we have received approval for all the links which we have added to this PR. Also, for all the GitHub repos, we have added a GSS disclaimer, which has been approved by GSS.

Reply:
Hi @lamek we have discussed this with the GSS and have added disclaimers right in the Prerequisites section wrt the same. This disclaimer was provided to us by the GSS. @RickJWagner @rkratky can you confirm, please?

Reply:
As discussed with @Preeticp and @vikram-redhat, I have removed the GSS Disclaimer from this assembly.